File: debug_daliuren.py
import sys
from kinliuren import Liuren

jieqi = "驚蟄"
cmonth = "五"
day_gz = "甲戌"
hour_gz = "庚子"

lr = Liuren(jieqi, cmonth, day_gz, hour_gz)
print("Sike:", lr.all_sike())
rel = lr.find_sike_relations()
print("Relations[0]:", rel[0])
print("Relations[2]:", rel[2])
print("Relations[7]:", rel[7])

print("zeike:", lr.zeike())
print("biyung:", lr.biyung())

res = lr.result(0)
print("Result Pattern:", res['格局'])

File: frontend/debug_daliuren.js
import { Liuren } from './src/engines/daliurenEngine.js';

const jieqi = "小雪";
const cmonth = "十二";
const day_gz = "辛卯";
const hour_gz = "甲戌";

const lr = new Liuren(jieqi, cmonth, day_gz, hour_gz);
console.log("Sike:", lr.all_sike());
const rel = lr.find_sike_relations();
console.log("Relations[0]:", rel[0]);
console.log("biyung:", JSON.stringify(lr.biyung()));
